### PR TITLE
docs(rxSortableColumn): Page objects to jsdoc3

### PR DESCRIPTION
**Pull request by Droogans**
### LGTMS
- [x] Dev LGTM
- [x] QE LGTM

### CODE REVIEW

File: src/components/rxSortableColumn/scripts/rxSortableColumn.page.js [hunk 1]
```diff
@@ -159,13 +209,14 @@ var rxSortableColumn = {
...
-       - Ascending sort:  (1)  means the arrow is pointed down. [0-9, a-z]
-       - Descending sort: (0)  means the arrow is pointed up.   [z-a, 9-0]
-       - Not sorted:     (-1)  means there is no arrow for this column.
-       @returns {Integer}: 1, 0, or -1 based on direction.
-       Use {@link encore.module:rxSortableColumn.sortDirections} when testing your columns.
+     * @instance
+     * @description The current sort direction of the column. Returns 1, 0, or -1 based on direction.
+     * - Ascending sort:  (1)  means the arrow is pointed down. [0-9, a-z]
+     * - Descending sort: (0)  means the arrow is pointed up.   [z-a, 9-0]
+     * - Not sorted:     (-1)  means there is no arrow for this column.
```
Comment (CITguy) [inline src/components/rxSortableColumn/scripts/rxSortableColumn.page.js]: Please update this description. The values are incorrect. (See [source](https://github.com/rackerlabs/encore-ui/blob/master/src/components/rxSortableColumn/scripts/rxSortableColumn.page.js#L8-L20))

```
- Ascending sort:   (1) means the arrow is pointed down. [0-9, a-z]
- Not Sorted:       (0) means there is no active arrow for this column.
- Descending sort: (-1) means the arrow is pointing up. [z-a, 9-0]
```

File: src/components/rxSortableColumn/scripts/rxSortableColumn.page.js [hunk 2]
```diff
@@ -20,7 +20,19 @@ var currentSortDirection = function (columnElement) {
...
+ * documentation below will assume that this example has been set up and executed.
+ * @example
+ * var myTable = {
+ *     get eleTableContainer() { return $('#my-table'); },
+ *
+ *     column: function (name) {
+ *         var column = this.eleTableContainer.element(by.cssContainingText($('rx-sortable-column'), name));
+ *         return encore.rxSortableColumn.initialize(column, 'item in repeaterString');
+ *     }
+ * };
```
Comment (CITguy) [inline src/components/rxSortableColumn/scripts/rxSortableColumn.page.js]: Can you please reduce the line lengths of this example? The code in the docs is scrolling out of view.

File: src/components/rxSortableColumn/scripts/rxSortableColumn.page.js [hunk 3]
```diff
@@ -173,10 +224,32 @@ var rxSortableColumn = {
...
+     * @description This exception is thrown whenever you attempt to access the data in a column's
+     * cells without specifying a repeater string for the functions {@link rxSortableColumn#data} and
+     * {@link rxSortableColumn#getDataUsing} to use. Including a `repeaterString` argument to
+     * {@link rxSortableColumn.initialize} is required in order to take advantage of this functionality.
+     * @example
+     * it('should throw an exception when trying to get column data', function () {
+     *     var repeaterString = undefined;
+     *     var column = encore.rxSortableColumn.initialize($$('rx-sortable-column').first(), repeaterString);
+     *     expect(column.data).to.eventually.eql(['Does', 'not', 'matter', 'exception', 'was', 'thrown']);
+     * });
```
Comment (CITguy) [inline src/components/rxSortableColumn/scripts/rxSortableColumn.page.js]: Can you please reduce the line lengths of this example? The code in the docs is scrolling out of view.

File: src/components/rxSortableColumn/scripts/rxSortableColumn.page.js (hunk 1 above)
Comment (Droogans) [inline src/components/rxSortableColumn/scripts/rxSortableColumn.page.js]: I'm confused why this was changed at all in https://github.com/rackerlabs/encore-ui/pull/1435

Was it not working with `-1` meaning "no sort"?

Comment (CITguy) [inline src/components/rxSortableColumn/scripts/rxSortableColumn.page.js]: The numbers were non-intuitive, with other languages when doing comparison you get `-1` for down (descending), `0` for no change, and `+1` for up (ascending).

**Comment by jj4th** [comment]
QE-LGTM pending addressing Ryan's comment

**Comment by CITguy** [comment]
LGTM

**Pull request merged**